.github/workflows/pre-commit.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     - uses: cvmfs-contrib/github-action-cvmfs@v3
     - uses: aidasoft/run-lcg-view@v4
       with:
-        release-platform: LCG_99/x86_64-centos7-clang10-opt
+        release-platform: LCG_104/x86_64-el9-clang16-opt
```

Comment:
I think in other places we use a Key4hep stack here, since we now have LLVM in there as well.

Reply:
But then it has to be changed because the run-lcg-view only works with LCG stacks

Reply:
Would something specific for pre-commit not work for EDM4hep? Because, e.g. podio uses a key4hep stack for the pre-commit workflow: 

https://github.com/AIDASoft/podio/blob/master/.github/workflows/pre-commit.yml

Reply:
Ah yes, I thought this was only used for LCG releases